docs/Configuration.md

```diff
@@ -470,6 +474,175 @@ Example configuration with example log can be found in [/examples/logs_json/](/e
 [filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.47.0/receiver/filelogreceiver
 [opentelemetry-log-collection]: https://github.com/open-telemetry/opentelemetry-log-collection
 
+## Setting source category
+
+For many Sumo Logic customers the [source category][source_category_docs] is a crucial piece of metadata
+that allows to differentiate data coming from different sources.
+This section describes how to configure the Sumo Logic Distribution of OpenTelemetry
+to decorate data with this attribute.
+
+To decorate all the data from the collector with the same source category,
```

Comment:
Maybe mention here that it's totally possible to have multiple exporters and set separate source categories for each of those.

Reply:
Hmm yes that's an option. I'm not sure though if we want to encourage this way.

There are many different ways to achieve the same goal, but this documentation definitely should not (and probably cannot) describe all of them.